contrib/runners/python_runner/tests/unit/test_pythonrunner.py

```diff
@@ -84,8 +86,13 @@ def test_action_returns_non_serializable_result(self):
         self.assertEqual(status, LIVEACTION_STATUS_SUCCEEDED)
         self.assertTrue(output is not None)
 
-        expected_result_re = (r"\[{'a': '1'}, {'h': 3, 'c': 2}, {'e': "
-                              "<non_simple_type.Test object at .*?>}\]")
+        if six.PY2:
+            expected_result_re = (r"\[{'a': '1'}, {'h': 3, 'c': 2}, {'e': "
+                                  "<non_simple_type.Test object at .*?>}\]")
+        else:
+            expected_result_re = (r"\[{'a': '1'}, {'c': 2, 'h': 3}, {'e': "
```

Comment:
What's the difference here other than the order of keys `c` and `h` in the second dict?

Reply:
Dictionary implementation in CPython changed, they are ordered by default now in Python 3.6/3.5, but having said that, relying in order is still a bad idea if you don't explicitly use `collections.OrderedDict` (in older versions, ordering is not guaranteed).

The problem here is that I can't make order-insensitive comparison, because the value is a string and not a dict...